lib/laa_crime_schemas/structs/amount.rb

```diff
@@ -8,6 +8,7 @@ class Amount < Base
 
       attribute? :metadata do
         attribute? :details, Types::String.optional
+        attribute? :case_reference, Types::String.optional
```

Comment:
I'm wondering if we should be making details a hash / object so that we don't add case_reference to this general object?

Reply:
`metadata` is already a hash/object. I was in 2 minds about this because like you say each arbitrary usage of `metadata` would lead to an update here too. Do we want to allow `metadata` to be completely arbitrary? In the end I thought adding the additional optional fields here serves as a basic form of documentation as to what kind of data to expect inside `metadata`